### PR TITLE
Organizer Contacts initially added

### PR DESCRIPTION
**Pull request by berlindave**
Done … hopefully according to all written and unwritten rules …

### CODE REVIEW

File: organizer-contacts.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Organizer Contacts
+### Name / e-mail / Twitter handle / Github / dewp.slack.com
+
+* André Brückner / andre.brueckner@posteo.de / [tichypress] (https://twitter.com/tichypress) / [tichypress] (https://github.com/tichypress) / tichypress
+* Alexander (Alex) Goller / alpipego@gmail.com / [alpipego] (https://twitter.com/alpipego) / [alpipego] (https://github.com/alpipego) / alpipego
+* David Stingl / mail@david-stingl.de / [berlindave] (https://twitter.com/berlindave) / [berlindave] (https://github.com/berlindave) / berlindave
+* Stefan Fröhlich / stefan@froehlich.it / ? / ? / –
+* Sven Brier / sb@netzgrad.org / [svenbrier] (https://twitter.com/svenbrier) / [svenbrier] (https://github.com/svenbrier) / svenbrier
+* Sven Hofmann / info@hofmannsven.com / [hofmannsven] (https://twitter.com/hofmannsven) / [hofmannsven] (https://github.com/hofmannsven) / hofmannsven
```
Comment (hofmannsven) [inline organizer-contacts.md]: Better remove the space between handle and link in [Markdown files](https://guides.github.com/features/mastering-markdown/#examples).

**Comment by hofmannsven** [comment]
Thanks for collecting all the contact information.
Are we good to go to add names and contact information of people that did not opt-in in #7?

File: organizer-contacts.md (hunk 1 above)
Comment (berlindave) [reply organizer-contacts.md]: Than you for the hint! Unfortunately it worked with the space in my Markdown editor.

**Comment by berlindave** [comment]
Good question, @hofmannsven ! The purpose of https://github.com/wp-berlin/planung/issues/7 stated by @alpipego was "Verteiler für Ankündigungen, Termine." Does a mailing list for announcements make sense only for people who opted in? How could we get them else before opting in?

**Comment by alpipego** [comment]
You're both correct.
Let's start with a list of organizers email addresses first, after opt-in (that's what `kein Zwang` was supposed to mean). The question whether we want to build attendees mailing list has to be discussed.

**Pull request merged**